demos/tests/run_tests.py

```diff
@@ -143,13 +157,29 @@ def option_to_args(key, value):
                     ' '.join(shlex.quote(str(arg)) for arg in case_args))
                 print(flush=True)
 
+                execution_time = -1
+                device = ""
+                pipline = []
+                for key , value in test_case.options.items():
```

Comment:
why comma is separated by spaces from both sides? better to write 'key, value'

Reply:
Done.